internal/database/migrations/sqlite/0002_denormalize_content.down.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN TRANSACTION;
+CREATE TABLE IF NOT EXISTS bookmark_temp(
```

Comment:
Why didn't you just `ALTER TABLE bookmark DROP COLUMN has_content;` ?

Reply:
I used workaround from old times when DROP COLUMN wasn't supported in sqlite. Hold a sec, I'll fix it.

Reply:
@fmartingr It still not supported, alas. So I leave this migration as-is.

Reply:
@Orhideous Support [was added in 3.35.0](https://www.sqlite.org/releaselog/3_35_0.html) and our library should support [up to 3.38.3](https://gitlab.com/cznic/sqlite/-/commit/61483fe9bc293b715fd476c7396b45f345544954). What error are you experiencing?

Reply:
Oh, that explains a lot. Focal Fossa distro still uses 3.31.1. Okay, thanks for clarification!

Reply:
Awesome, let's go!